### PR TITLE
git clone only if LIGHTUP_DEBUG is set

### PR DESCRIPTION
**Pull request by manubansal**

### CODE REVIEW

File: init.sh [hunk 1]
```diff
@@ -78,18 +80,20 @@ if [[ ! -f lup-${NAME} ]]; then
...
+if [[ -z "${LIGHTUP_DEBUG}" ]]; then
+    # default production path
+    echo "downloading lightup manager"
+    LIGHTUP_TAR_GZ_TARGET="v${BRANCH}.tar.gz"
     curl -H 'Cache-Control: no-cache' -L https://github.com/lupfoss/lupmgr/archive/refs/tags/"${LIGHTUP_TAR_GZ_TARGET}" --output lupmgr.tar.gz
     tar -xvf lupmgr.tar.gz
     [[ -d lupmgr ]] && rm -rf lupmgr
-    mv lupmgr-"${LIGHTUP_TAR_GZ_VERSION}" lupmgr
+    mv lupmgr-"${BRANCH}" lupmgr
     cd lupmgr
```
Comment (rajivramanathan) [inline init.sh:91]: @manubansal @jeff-lightup I think there is another problem that I think needs to be tackled. We store all our keys in the lupmgr directory. If we delete the lupmgr directory, we will delete the keys. Do we lose something in the process? 

Also, if we do delete the keys, is there some "remnants" that we will need to clean up from the connect server?

Comment (rajivramanathan) [reply init.sh:91]: Also, I don't know if we should overload BRANCH. TAG is not a BRANCH and the BRANCH.tar.gz will not work unless BRANCH is a tag.

**Comment by jeff-lightup** [comment]
I think that PR https://github.com/lupfoss/lupmgr/pull/35 covers the same workflow that is in this PR.

**Comment by jeff-lightup** [comment]
Closing per discussion [here](https://lightupai.slack.com/archives/CHWBVK0R0/p1644951118890269):

> yeah, i think mine can be closed without merging

**Pull request closed without merging**